geotrek/trekking/models.py

```diff
@@ -149,33 +147,36 @@ class Trek(Topology, StructureRelated, PicturesMixin, PublishableMixin, MapEntit
     practice = models.ForeignKey('Practice', related_name="treks", on_delete=models.CASCADE,
                                  blank=True, null=True, verbose_name=_("Practice"))
     accessibilities = models.ManyToManyField('Accessibility', related_name="treks", blank=True,
-                                             verbose_name=_("Accessibility"))
-    accessibility_level = models.ForeignKey('AccessibilityLevel', related_name="treks", blank=True,
-                                            verbose_name=_("Level"), null=True, on_delete=models.PROTECT,
-                                            help_text=_("Beginner (Little drop – terrain without difficulties) / Experienced "
-                                                        "(Significant slope – Technical terrain, with obstacles)"))
-    accessibility_slope = models.TextField(verbose_name=_("Slope"),
-                                           blank=True, help_text=_("Description of the slope: greater than 10% "
-                                                                   "(Requires assistance when the slope is greater "
-                                                                   "than 8%); slope break"))
-    accessibility_covering = models.TextField(verbose_name=_("Covering"),
+                                             verbose_name=_("Type accessibility"))
```

Comment:
Je dirais soit "Accessibility type", soit "Type of accessibility"

Reply:
Oui